rotkehlchen/chain/ethereum/modules/thegraph/decoder.py

```diff
@@ -0,0 +1,187 @@
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_stake_delegated(self, context: DecoderContext) -> DecodingOutput:
+        deposit_event, burn_event = None, None
+        delegator = hex_or_bytes_to_address(context.tx_log.topics[2])
+        if delegator is None or self.base.is_tracked(delegator) is False:
+            return DEFAULT_DECODING_OUTPUT
+        indexer = hex_or_bytes_to_address(context.tx_log.topics[1])
+        stake_amount = hex_or_bytes_to_int(context.tx_log.data[:32])
+        stake_amount_norm = token_normalized_value(
+            token_amount=stake_amount,
+            token=self.token,
+        )
+        # identify and override the original stake Transfer event
+        for event in context.decoded_events:
+            if (
+                event.location_label == delegator and
+                event.address == CONTRACT_THEGRAPH_STAKING and
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE
+            ):
+                initial_amount_norm = event.balance.amount
+                event.event_type = HistoryEventType.STAKING
+                event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                event.balance = Balance(amount=stake_amount_norm)
+                event.notes = f'Delegate {stake_amount_norm} GRT to indexer {indexer}'
+                event.counterparty = CPT_THEGRAPH
+                deposit_event = event
+
+                # also account for the GRT burnt due to delegation tax
+                tokens_burnt = initial_amount_norm - stake_amount_norm
+                if tokens_burnt > 0:
+                    burn_event = self.base.make_event_from_transaction(
+                        transaction=context.transaction,
+                        tx_log=context.tx_log,
+                        event_type=HistoryEventType.SPEND,
+                        event_subtype=HistoryEventSubType.FEE,
+                        asset=A_GRT,
+                        balance=Balance(amount=tokens_burnt),
+                        location_label=delegator,
+                        notes=f'Burn {tokens_burnt} GRT as delegation tax',
+                        counterparty=CPT_THEGRAPH,
+                        address=context.tx_log.address,
+                    )
+                    maybe_reshuffle_events(
+                        ordered_events=[deposit_event, burn_event],
+                        events_list=context.decoded_events,
+                    )
+                    return DecodingOutput(event=burn_event)
+                break
+
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_stake_locked(self, context: DecoderContext) -> DecodingOutput:
+        delegator = hex_or_bytes_to_address(context.tx_log.topics[2])
+        if delegator is None or self.base.is_tracked(delegator) is False:
+            return DEFAULT_DECODING_OUTPUT
+        indexer = hex_or_bytes_to_address(context.tx_log.topics[1])
```

Comment:
Use newline after if. Helps readability

```suggestion
        indexer = hex_or_bytes_to_address(context.tx_log.topics[1])

```